src/main/java/org/gridsuite/merge/orchestrator/server/repositories/ProcessConfigEntity.java

```diff
@@ -7,29 +7,44 @@
 package org.gridsuite.merge.orchestrator.server.repositories;
 
 import lombok.Getter;
-import org.springframework.data.cassandra.core.mapping.PrimaryKey;
-import org.springframework.data.cassandra.core.mapping.Table;
-
+import lombok.NoArgsConstructor;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.ForeignKey;
+import javax.persistence.Id;
+import javax.persistence.Index;
+import javax.persistence.Table;
 import java.util.List;
 import java.util.UUID;
 
 /**
- * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com
  * @author Franck Lecuyer <franck.lecuyer at rte-france.com
  */
+@NoArgsConstructor
 @Getter
-@Table("configs")
+@Entity
+@Table(name = "configs")
 public class ProcessConfigEntity {
 
-    @PrimaryKey
+    @Id
+    @Column(name = "processUuid")
     private UUID processUuid;
 
+    @Column(name = "process")
     private String process;
 
+    @Column(name = "businessProcess")
     private String businessProcess;
 
+    @Column(name = "tso")
+    @ElementCollection
+    @CollectionTable(foreignKey = @ForeignKey(name = "processConfigEntity_tsos_fk"), indexes = {@Index(name = "processConfigEntity_tsos_idx", columnList = "ProcessConfigEntity_processUuid")})
```

Comment:
why you add index for this field?

Reply:
To avoid sequential scans if possible on select request.
Things similar in actions-server with the countries in the FilterContingencyListEntity